ui/perfherder/alerts/StatusDropdown.jsx

```diff
@@ -257,38 +258,98 @@ export default class StatusDropdown extends React.Component {
     alertStatus === 'investigating' ||
     (alertStatus !== status && this.isResolved(alertStatus));
 
+  getRemainingTime(firstDate, secondDate) {
+    const differenceInTime = Math.abs(secondDate - firstDate);
+    let differenceInDays = Math.ceil(differenceInTime / (1000 * 60 * 60 * 24));
+
+    // If weekday of date1 is bigger than date2 then it is a new week
+    // and we have to subtract the weekend days
+    if (firstDate.getUTCDay() > secondDate.getUTCDay()) {
```

Comment:
```suggestion
    const dueDateIsNextWeek = firstDate.getUTCDay() > secondDate.getUTCDay();  // this can also be moved in another method 
    if (dueDateIsNextWeek) {
```

Reply:
This was not addressed yet. If addressed it will make possible to remove unnecessary comments.